### PR TITLE
ZK-3733: listbox with listgroup causes ie11 long running script warning

### PR DESCRIPTION
**Pull request by charlesqqq**

### CODE REVIEW

File: zk/src/archive/web/js/zk/au.js [hunk 1]
```diff
@@ -1642,10 +1642,17 @@ zAu.cmd1 = /*prototype*/ {
 	addChd: function (wgt) {
 		if (wgt) {
 			var fn = function (child) {
-				var act = _beforeAction(child, 'show');
+				var act = _beforeAction(child, 'show'),
```
Comment (DevChu) [inline zk/src/archive/web/js/zk/au.js]: should consider a change in addChd function flow, other component also could do fireSizedLater, not only Listbox

Comment (charlesqqq) [reply zk/src/archive/web/js/zk/au.js]: ok

**Comment by DevChu** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**